cfgov/jinja2/v1/_includes/organisms/filterable-list.html

```diff
@@ -83,29 +81,27 @@
             {% endif %}
         {% endfor %}
 
-        {% if 'filterable-list' in value.form_type and posts is defined %}
-            {% if form.is_valid() %}
-                {% set count = posts.paginator.count %}
-                {% if count == 0 %}
-                    {# The if statement here allows access to the
-                        notification_opts variable outside of the for loop.
-                    #}
-                    {% if notification_opts.update({
-                        'type': 'warning',
-                        'is_visible': true,
-                        'message': 'Sorry, there were no results based on your filter selections.',
-                        'explanation': 'Please reset the filter or change your selections and try again.'
-                    }) %}{% endif %}
-                {% else %}
-                    {# The if statement here allows access to the
-                        notification_opts variable outside of the for loop.
-                    #}
-                    {% if notification_opts.update({
-                        'type': 'success',
-                        'is_visible': true,
-                        'message': count ~ ' filtered result' ~ count | pluralize('s')
-                    }) %}{% endif %}
-                {% endif %}
+        {% if posts and form.is_valid() %}
```

Comment:
Looks like this should be `{% if posts is defined and form.is_valid() %}` otherwise http://localhost:8000/about-us/newsroom/?categories=blog&topics=financial-education&authors=holly-petraeus&from_date=2017 doesn't show the warning message.

Reply:
Changed to more simply `{% if form.is_valid() %}` in 33319d5; `posts` should always be defined because `filter_data['page_set']` is always defined [here](https://github.com/cfpb/cfgov-refresh/blob/ff8f89f3ca29582ee3504ffb9d9086e6bef6d86c/cfgov/v1/util/filterable_list.py#L77).